src/main/scala/org/broadinstitute/dsde/agora/server/webservice/methods/MethodsQueryHandler.scala

```diff
@@ -30,4 +32,9 @@ class MethodsQueryHandler extends Actor {
       case Some(method) => context.parent ! grater[AgoraEntity].toJSON(method)
     }
   }
+
+  def query(requestContext: RequestContext, agoraSearch: AgoraSearch): Unit = {
+    val entities = AgoraDao.createAgoraDao.find(AgoraEntity.fromAgoraSearch(agoraSearch))
```

Comment:
As a side note I would be curious to stress test this. We need to consider that creating a new agora dao each time may not be the best practice. (Need to do some research in to mongo best practices)